test/integration/parachain/staging/sequential/loans.test.ts

```diff
@@ -373,6 +371,12 @@ describe("Loans", () => {
         });
     });
 
+    describe("getAccruedRewardsOfAccount", () => {
+        it("should return correct amount of reward", () => {
```

Comment:
We should check if this test is needed as integration test.
We might get away with just testing our calcs as unit tests instead.

Reply:
Hmm true, it just returns value from the parachain so no calculations to test on that front :+1: